Fix #6796: Skip setTimeout if Menubar delay is 0

JHookDev opened the pull request:
To fix issue 6796 I am proposing that we skip using setTimeout if the Menubar delay is 0. The submenu show will execute immediately rather than being submitted to the browser message queue for processing.

JHookDev committed:
Merge pull request #1 from primefaces/master

Update Fork from master

JHookDev committed:
Fix #6796: Skip setTimeout if delay is 0

Bypassing the setTimeout call if the menubar delay is 0. The submenu show will execute immediately rather than being submitted to the browser message queue for processing.

melloware commented on src/main/resources/META-INF/resources/primefaces/menu/menu.menubar.js:
```diff
@@ -43,11 +43,18 @@ PrimeFaces.widget.Menubar = PrimeFaces.widget.TieredMenu.extend({
-                  .position(pos)
-        }, this.cfg.delay);
+        //avoid using timeout if delay is 0
+        if(this.cfg.delay != null && this.cfg.delay > 0) {
```
You can change this to...
```javascript
if(this.cfg.delay && this.cfg.delay > 0) {
```

This will be false if delay ==0 or delay == null or delay == undefined.

JHookDev replied:
Thanks for the feedback. I've updated the change accordingly.

JHookDev committed:
Fix #6796: Skip setTimeout if delay is 0

modified condition for this.cfg.delay removing '!= null', so that it checks for null and undefined. The check for '> 0' will still be intact.
